lib/faraday/deprecate.rb

```diff
@@ -54,6 +54,7 @@ def self.skip # :nodoc:
         else :skip
         end
       end
+      @skip = 1
```

Comment:
Just noting this line here, it's for debugging while writing this PR, I take it. Should it be removed?

Reply:
Yes, the line is for only debugging. I'll remove this soon.